resources/service-catalog/charts/etcd-stateful/templates/03-statefulset.yaml

```diff
@@ -137,6 +137,7 @@ spec:
                 --initial-cluster-token etcd-cluster-1 \
                 --initial-cluster ${PEERS} \
                 --initial-cluster-state ${STATE} \
+                --snapshot-count=10000 \
```

Comment:
as discussed, please extract it to the variable and add comment with link to our issue on GitHub. Thanks to that this value won't be so magic